ci-operator/step-registry/ipi/conf/aws/user-permissions/ipi-conf-aws-user-permissions-ref.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "ipi/conf/aws/min-perms-user/ipi-conf-aws-min-perms-user-ref.yaml",
+	"path": "ipi/conf/aws/user-permissions/ipi-conf-aws-user-permissions-ref.yaml",
```

Comment:
shall we continue to naming the step with some `min-perms` to indicate it is used for users with minimal permission ?

Reply:
Done